src/app/shared/entry.ts

```diff
@@ -259,11 +273,21 @@ export abstract class Entry implements OnInit, OnDestroy {
    * @param {Tag|WorkflowVersion} b - version b
    * @returns {number} - indicates order
    */
-  entryVersionSorting(a: Tag | WorkflowVersion, b: Tag | WorkflowVersion): number {
+
+  verifiedSorting(a: Tag | WorkflowVersion, b: Tag | WorkflowVersion): number {
     if (a.verified && !b.verified) {
       return -1;
     } else if (!a.verified && b.verified) {
       return 1;
+    } else {
+      return 0;
+    }
+  }
+  workflowVersionSorting(a: WorkflowVersion, b: WorkflowVersion): number {
+
+    const verifiedSorting = this.verifiedSorting(a, b);
+    if (verifiedSorting !== -2) {
```

Comment:
Since you got rid of `-2` in `verifiedSorting`, I think you should be checking against `0`.